src/app/join/components/JoinLayout/JoinLayout.tsx

```
@@ -0,0 +1,22 @@
+import { PropsWithChildren } from 'react';
+
+import { contentContainer } from '../../JoinPage.css';
+import FormGuard from '../FormGuard/FormGuard';
+import JoinTitleSection from '../JoinTitleSection/JoinTitleSection';
+
+import Logo from '@/components/Logo/Logo';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)